tools/istio-docker.mk

```diff
@@ -79,9 +79,7 @@ docker.proxyv2: install/gcp/bootstrap/gcp_envoy_bootstrap.json
 docker.proxyv2: $(ISTIO_ENVOY_LINUX_RELEASE_DIR)/envoy
 docker.proxyv2: $(ISTIO_OUT_LINUX)/pilot-agent
 docker.proxyv2: pilot/docker/Dockerfile.proxyv2
-docker.proxyv2: pilot/docker/envoy_pilot.yaml.tmpl
 docker.proxyv2: pilot/docker/envoy_policy.yaml.tmpl
```

Comment:
Why not clean up policy one, too? Telemetry and policy bootstraps should be next to each other.

Reply:
Policy is still using this for whatever reason. I don't know why, they clearly should be consistent. I don't want to do that in this PR though. I can keep the pilot/telemetry file around until then, but they are completely broken and not used which is why I removed them

Reply:
We had a test for internal telemetry, so it should be still working, right? We have to keep mixer features until 1.7 since it's still supported.

Reply:
Yes, but pilot/docker/envoy_telemetry.yaml.tmpl is never used, because we are using the file in manifests/ I linked about. This is dead code I am removing, nothing more

Reply:
Ok. In any case, you don't need my approval for this PR.